.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Critical? | no
```

Comment:
I also think that this line should be deleted. From a user perspective every tiny bug is immediately critical, because it is affecting them directly (otherwise they won't report anyway).